src/ui/layerButton.css

```
@@ -78,21 +109,21 @@
 /* Checked-while-hidden: checked fill replaces the hatch; only the border is #0066CC. */
 .nlb.nlb-hidden.nlb-checked {
   background: var(--nlb-checked, var(--nlb-fill));
-  border: 2px solid #0066CC;
+  --nlb-border: 2px solid #0066CC;
 }
 
 /* ---- shadow_only: dashed gray border (three states) ---- */
 .nlb.nlb-shadow-only {
-  border: 2px dashed rgba(128, 128, 128, 0.8);
+  --nlb-border: 2px dashed rgba(128, 128, 128, 0.8);
   background: var(--nlb-fill);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)